src/getDataSource.js

```diff
@@ -11,7 +11,7 @@ const getDataSource = ( opts ) => {
     return prismic({
       'url': opts.dataSource.url,
       'accessToken': opts.dataSource.accessToken,
-      'linkResolver': function( ctx, doc ) {
+      'linkResolver': opts.config.linkResolver || function( ctx, doc ) {
```

Comment:
Do you think we should put a check in place to make sure that this comes through as a function like we are expecting, otherwise the prismic API script would fail and people may not be aware why?

Would be nice if by this point it was just the variable that got passed in and if the check fails then set it to use the default function instead or an `||` , this does work currently fine however.